.github/workflows/build.yml

```
@@ -28,6 +28,10 @@ jobs:
         - name: Fix iOS build bug
           if: matrix.target == 'ios'
           run: rm ios/Podfile
+        - name: Pub Get & Build Runner
+          run: |
+            flutter pub get
+            flutter pub run build_runner build
```

```suggestion
            flutter pub run build_runner build --delete-conflicting-files
```

In case, someone did commit auto-generated files?

---

I'd rather have the CI build fail in this case so we know we have to fix it. WDYT?

---

I don't have any strong preference. If it doesn't work like we expected, we can still change it later 👍